wandb/sdk/wandb_run.py

```diff
@@ -2194,6 +2206,7 @@ def finish(
         - Crashed: Run that stopped sending heartbeats unexpectedly.
         - Finished: Run completed successfully (`exit_code=0`) with all data synced.
         - Failed: Run completed with errors (`exit_code!=0`).
+        - Killed: Run was forcibly stopped before it could finish.
```

Comment:
Where does this come from?

Reply:
I got it from here: https://docs.wandb.ai/guides/runs/#run-states